src/ui/server/test-mocks/mock-application.ts

```diff
@@ -102,6 +102,11 @@ export const mockExportContract = {
   agent: mockExportContractAgent,
 };
 
+export const mockOtherAwardMethod = {
+  awardMethod: EXPORT_CONTRACT_AWARD_METHOD.OTHER.DB_ID,
+  otherAwardMethod: 'test',
```

Comment:
"Mock other award method"?

Reply:
Data structure is misleading here, not alinged to the DB like the other mocks, `otherAwardMethod` is a single string field, we could just update `mockExportContract` to have aa `otherAwardMethod` property?